project-backend/Providers/JobProvider/JobProvider.cs

```diff
@@ -33,6 +34,25 @@ public JobDAO AddJob(string name, string description, int userId)
             return newJob;
         }
 
+        public JobDAO EditJob(int id, string newName, string newDescription, int userId)
+        {
+            JobDAO correspondingJob = _dbContext.Jobs.Find(id);
```

Comment:
```suggestion
            JobDAO correspondingJob = _dbContext.Jobs
                .Include(job => job.Images)
                .Where(job => job.Id == id)
                .FirstOrDefault();
```
Because the job response object contains image URLs, we need to include the images here